dashboard/src/actions/apps.ts

```diff
@@ -31,11 +31,14 @@ const allActions = [requestApps, receiveApps, errorApps, errorDeleteApp, selectA
 );
 export type AppsAction = typeof allActions[number];
 
-export function getApp(releaseName: string, namespace: string) {
+export function getApp(hrName: string, releaseName: string, namespace: string) {
```

Comment:
I found the variable names to be a bit confusing. what do you think about refactoring  the variable names to `crdName` and `releaseName`? /cc @prydonius